packages/design-system/src/components/Dropdown/Dropdown.tsx

```diff
@@ -44,6 +43,7 @@ const Dropdown = forwardRef(
 			animated: 250,
 			gutter: 4,
 			loop: true,
+			baseId: useId().id,
```

Comment:
Wouldn't that be different each time as well? If you need to set a definite id for your snapshots, wouldn't we need this?  

```suggestion
			baseId: useId().id || props.id,
```

Reply:
yes it would, but we already mock `useId` in tests to return a fixed number. 
And I wanted to prevent exposition of new properties if not necessary.